src/cli.rs

```diff
@@ -681,6 +689,26 @@ pub struct GcArgs {
     pub clean: bool,
 }
 
+lazy_static::lazy_static! {
+    static ref RENEW_EXPIRING_HELP: String = format!("Renew all wildcard audits which will have expired {WILDCARD_AUDIT_EXPIRATION_STRING} from now.");
+}
+
```

Comment:
Agreed, though printing help output does plenty of runtime string formatting. I could just put the words inline, at the cost of any changes to that duration being error-prone.

Reply:
Yeah, I suppose it'd technically be possible to use `concat!(...)` or something like that, but it's pretty gross.

Reply:
I'm going to change this to a doc string, because our other docs also have it non-programmatic anyway. It'd be nice if `clap` supported a `std::fmt::Arguments` for the help so you don't pay the price just to create a string that generally won't be used often.